tests/integration/parallel_skip.yaml

```
@@ -28,23 +28,27 @@
 - test_merge_tree_s3/
 - test_mysql_database_engine/
 - test_parts_delete_zookeeper/
+- test_postgresql_replica_database_engine/
 - test_profile_max_sessions_for_user/
 - test_quorum_inserts_parallel/
 - test_random_inserts/
 - test_replace_partition/
 - test_replicated_database/
 - test_replicated_fetches_timeouts/
 - test_replicated_merge_tree_wait_on_shutdown/
+- test_server_overload/
```

```suggestion
- test_server_overload/
```

There is no such test in 25.3